### PR TITLE
Merged ACheckerVillagantol to Development

### PR DESCRIPTION
**Pull request by TedskieBear**
Submitted activity.

### CODE REVIEW

File: src/AChecker.cpp [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #include "AChecker.h"
 
+ char checkMyLetter(char inputChar) { // 'override' is optional
+        std::cout << "YAY\n";
```
Comment (amstelventanilla) [inline src/AChecker.cpp]: remove unnecessary comments

**Review by amstelventanilla — CHANGES_REQUESTED** [review]
review your code

**Review by amstelventanilla — APPROVED** [review]
good job!

**Comment by densioli** [comment]
closing this since it is old and will not be used

**Pull request closed without merging**